docs/sites.yml

```diff
@@ -10731,3 +10731,15 @@
     - Food
   built_by: Ramón Chancay
   built_by_url: https://ramonchancay.me/
+- title: Wanaboat.fr
+  main_url: https://www.wanaboat.fr
+  url: https://www.wanaboat.fr
+  description: >
+    This a boating classifieds website. It presents dinghys, catamarans and anything that floats or goes on the water and is for sale in France and Europe.
+  categories:
+    - Directory
+    - CMS:Contentful
```

Comment:
```suggestion
```

This tag is actually used for starters in the starter showcase, could you remove it so the only tags are for those in the site showcase? 🙂